components/threading/global_threads/thread_list/thread_list.tsx

```diff
@@ -116,7 +116,11 @@ const ThreadList = ({
     }, [setFilter]);
 
     const handleLoadMoreItems = useCallback(async (startIndex) => {
-        const before = data[startIndex - 1];
+        let before = data[startIndex - 1];
+
+        if (before === selectedThreadId) {
+            before = data[startIndex - 2];
+        }
```

Comment:
`selectedThreadId` is a thread that might be in the list only because it's
selected, so let's don't take into account this one when we paginate for
results.